_plugins/values.rb

```diff
@@ -67,5 +67,14 @@ def gen_values(versions, imageNames, imageRegistry)
     flexvol:
       image: #{imageRegistry}#{imageNames.fetch("flexvol")}
       tag: #{versions.fetch("flexvol")}
+    # Configuration for the tigera operator
+    tigeraOperator:
```

Comment:
This could be OK if we're cutting corners, but to be correct we should add an if statement to this file that populates different values data depending on which chart it's being rendered for. Maybe you leave this as is and I clean it up in the helm work I'm planning to do next week?